src/components/ceremony/handlers/delete-engagement-default-ceremony.handler.ts

```diff
@@ -17,6 +17,10 @@ export class DetachEngagementRootDirectoryHandler
       return;
     }
 
+    if (engagement.changeset) {
```

Comment:
Could the changeset just be passed though here to the ceremony delete? I know we still need to get ceremonies completely changeset aware, so if not can you leave a todo stating this is temporary until then?